### ISSUE TITLE
global is not defined

### ISSUE THREAD

**Issue by ralyodio** [torusresearch/OpenLoginSdk#110]
Trying to use in sveltekit:

```
import { browser } from '$app/env';
import OpenLogin from '$lib/auth/openLogin';

class Chat {
	constructor(user) {
		this.user = user;
		this.chat = null;

		if (browser) {
			this.auth = new OpenLogin(user);
			this.chat = matrixcs.createClient('https://matrix.org');
		}
	}

	async login() {
		return await this.auth.login();
	}

	async getRooms() {
		try {
			return (await this.chat.publicRooms()).chunk;
		} catch (err) {
			console.error(err);
			return [];
		}
	}
}

export { Chat };
```

```
import Auth from './auth.ts';
import OpenLogin from '@toruslabs/openlogin';
import { Ed25519KeyIdentity } from '@dfinity/identity';
import { identity, currentAccount } from '$stores/auth';
import { fromHexString } from '$lib/helpers.ts';
import { browser } from '$app/env';

class Open extends Auth {
	constructor() {
		super();
		this.openlogin = null;
		this.t = browser && localStorage.getItem('_loginType');
	}

	async login() {
		this.openlogin = await this.load();
		/*
		if (this.openlogin.privKey) {
			await this.openlogin.logout();
		}
*/
		await this.openlogin.login();
		const id = Ed25519KeyIdentity.generate(new Uint8Array(fromHexString(this.openlogin.privKey)));
		console.log('id: ', id);
		if (id) {
			identity(id);
			currentAccount(0);
			browser && localStorage.setItem('_loginType', t);
		} else {
			throw new Error('Failed to connect to your wallet');
		}
	}

	async load() {
		if (!this.openlogin) {
			this.openlogin = new OpenLogin({
				clientId:
					'xxx',
				network: 'mainnet',
				uxMode: 'popup'
			});
		}
		await this.openlogin.init();
		return this.openlogin;
	}
}

export default Open;
```

**Comment by ralyodio** [comment]
Hers' the error:

```
global is not defined

node_modules/@toruslabs/eccrypto/browser.js@http://localhost:3000/node_modules/.vite/@toruslabs_openlogin.js?v=19012050:246:25
__require2@http://localhost:3000/node_modules/.vite/chunk-6DBBYCL5.js?v=19012050:19:50
@http://localhost:3000/node_modules/.vite/@toruslabs_openlogin.js?v=19012050:5316:31

````

**Comment by chaitanyapotti** [comment]
Fixed in v2.0.0 Please upgrade
